pymove/core/__init__.py

```diff
@@ -1,3 +1,9 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5539342)